FirebaseAppCheck/Sources/DebugProvider/API/FIRAppCheckDebugProviderAPIService.m

```diff
@@ -82,7 +82,7 @@ - (instancetype)initWithAPIService:(id<FIRAppCheckAPIServiceProtocol>)APIService
       });
 }
 
-#pragma mark -
+#pragma mark - Helpers
```

Comment:
Not sure if this was intended to be labeled or not.

Reply:
LGTM